index.js

```diff
@@ -264,7 +269,7 @@ window.mobileCheck = function() {
                     }
                 })
             }
-            centreData = Object.assign({}, {name:centre["name"]}, daysData);
+            centreData = Object.assign({}, {name:centre["name"], name_district:centre["name"]+" ("+centre["district"]+")"}, daysData);
```

Comment:
```suggestion
            centreData = Object.assign({}, {name:{name:centre["name"], district:centre["district"]}, daysData);
```

Here instead of creating string we are just storing info which we will use in "render:" for constructing the string